molecule/logger.py

```diff
@@ -19,11 +19,27 @@
 #  DEALINGS IN THE SOFTWARE.
 
 import logging
+import os
 import sys
 
 import colorama
 
-colorama.init(autoreset=True)
+
+def should_do_markup(file=sys.stdout):
+    if os.environ.get('PY_COLORS') == '1':
```

Comment:
There's a prior art on boolean detection you may want to reuse:
```python
from ansible.module_utils.parsing.convert_bool import boolean as to_bool
...
is_py_colors_enabled = to_bool(os.environ.get('PY_COLORS'), strict=False)
```

* https://github.com/ansible/ansible/blob/c37f6bed1c44f7d6ef854baec4c5e04a037c048f/lib/ansible/module_utils/parsing/convert_bool.py
* https://github.com/ansible/ansible/blob/c37f6bed1c44f7d6ef854baec4c5e04a037c048f/lib/ansible/plugins/filter/core.py#L89

Reply:
Sure, I will use to_bool, funny I was considering doing it.